src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerLinkCase.java

```diff
@@ -41,11 +41,15 @@ public class CaseworkerLinkCase implements CCDConfig<CaseData, State, UserRole>
     @Value("${feature.link-case.enabled}")
     private boolean linkCaseEnabled;
 
+    private final CaseLinkedNotification caseLinkedNotification;
```

Comment:
Autowire this field (might need to remove the final keyword if Autowiring a final field isn't allowed)